### PR TITLE
[doc] Added metadata_column parameter usage example in mysql cdc action

### PR DESCRIPTION
**Pull request by zhuangchong**
### Purpose

Added metadata_column parameter usage example in mysql cdc action

Before modification: 
In mysql database cdc, the metadata column is separated by commas; 
In mysql table cdc, the metadata column directly obtains multiple parameters.

### CODE REVIEW

File: paimon-flink/paimon-flink-cdc/src/main/java/org/apache/paimon/flink/action/cdc/CdcActionCommonUtils.java [hunk 1]
```diff
@@ -61,7 +61,7 @@ public class CdcActionCommonUtils {
     public static final String PARTITION_KEYS = "partition_keys";
     public static final String PRIMARY_KEYS = "primary_keys";
     public static final String COMPUTED_COLUMN = "computed_column";
-    public static final String METADATA_COLUMN = "metadata_column";
+    public static final String METADATA_COLUMNS = "metadata_columns";
```
Comment (JingsongLi) [inline paimon-flink/paimon-flink-cdc/src/main/java/org/apache/paimon/flink/action/cdc/CdcActionCommonUtils.java:64]: Are you change API?

Comment (zhuangchong) [reply paimon-flink/paimon-flink-cdc/src/main/java/org/apache/paimon/flink/action/cdc/CdcActionCommonUtils.java]: done.

**Comment by JingsongLi** [comment]
API is something we need to focus on considering compatibility, and behavior is a part of API. Unless absolutely necessary, we should not compromise API compatibility.

**Review by JingsongLi — APPROVED** [review]
+1

**Pull request merged**